src/common/generator.js

```diff
@@ -21,9 +21,21 @@ window.POG=(function() {
             else {
                 selector += '[' + name + '=\'' + value + '\']';
             }
-            if (document.querySelectorAll(selector).length === 1) {
-                response = selector;
-            }
+	    //Surrounding with try to catch the exceptions that occur and crash the program on
+	    //certain web pages. 
+	    try
+	    {
+		if (document.querySelectorAll(selector).length === 1) {
+                    response = selector;
+		}
+	    }
+	    catch (e) {
+		
+		console.log(e);
+		console.log("Bad selector: " + selector);
```

Comment:
`console.log` stuff is not a ideal thing to use.

maybe you could use the notify module to display what was the error that occurred, rather than hiding it under the rug(inside the dev-tools)